### PR TITLE
Final branch

### PR DESCRIPTION
**Pull request by MarkoKermi**
- [ ] I fetched the countries data from https://restcountries.com/

- [ ] In the home page you can find a cards with all countries and also a dropdown menu where you can select the continents. 
- [ ] When you click the cards, a new detail page is showing where you can see the details of all countries (population, flag, capitol city, area and time zone.)
- [ ] You can see the walkthrough video of the web app [here]( https://www.loom.com/share/abaab4ffdd0c424686cac82fc5c3483e).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,43 +163,31 @@ Example:
...
+- **How to report bugs?**
 
-  - [Answer_1]
+  - Please raise [github issue](https://github.com/MarkoKermi/metrics_webapp/issue) with more details about the bug.
 
-- **[Question_2]**
+- **What is the licensing policy of the project?**
 
-  - [Answer_2]
+  - The project is MIT licensed, please refer to the License section
```
Comment (VICTOR-CHUKWU) [inline README.md:190]: - [ ] your MIT  link is broken, kindly fix it

File: README.md [hunk 2]
```diff
@@ -60,174 +32,128 @@ After you're finished please remove all the comments and instructions!
...
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- See it live by clicking [Live Demo Link](https://countries-web-app.onrender.com)
 
-- [Live Demo Link](https://google.com)
+- I have added a [Video Presentaition](https://www.loom.com/share/0fd72d02c23b4c6e8563e82909a55463)
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] while showing the piece of code you are proud of, your screen was stuck to your browser not showing your vscode, kindly add a screen showing your vscode while describing the piece of code your are proud of

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @MarkoKermi , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md:190]: This change has been done well

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @MarkoKermi, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**